cognite/seismic/protos/query_service_messages.proto

```diff
@@ -16,11 +16,13 @@ message SurveyQueryRequest {
     bool list_files = 2;          // set to true to list the survey files in the response (default: false)
     bool include_metadata = 3;    // set to true to include metadata in the response (default: false)
     CoverageParameters include_coverage = 4;    // set this field to include coverage in the response (default: false)
+    bool include_grid_transformation = 5; // set to true to include the grid transformation in the response, if available (default: false)
```

Comment:
Is "grid transformation" the right name here or is "grid definition" or something better? @ElkaSierra , what do you call the data about how the bin grid is laid out on the surface of the earth?

Reply:
Well, we already called it a grid_transformation in the `RegisterSurveyRequest` and `EditSurveyRequest`. Those can be renamed, of course.